### PR TITLE
readme: Update AWS PV provisioner support statement, fix delete command

### PR DESCRIPTION
**Pull request by lachie83**
Given investigation in #341 AWS ebs provisioner is supported.

### CODE REVIEW

**Comment by rimusz** [comment]
@lachie83 'We are planning to support AWS(kubernetes.io/aws-ebs) soon.' should be removed from readme.

**Comment by hongchaodeng** [comment]
LGTM after fixing the comments mentioned above. Thanks @rimusz, @lachie83 !

File: README.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ NAME                     TYPE
 etcd-operator-backup   kubernetes.io/gce-pd
 ```
 
-This is used to request the persistent volume to store the backup data. (TODO: We are planning to support AWS EBS soon.)
+This is used to request the persistent volume to store the backup data. (AWS EBS supported using --pv-provisioner=kubernetes.io/aws-ebs)
```
Comment (xiang90) [inline README.md]: link to deployment file? tell users exactly where to change?

Comment (xiang90) [inline README.md]: ideally we can create a deployment-aws.yaml

Comment (lachie83) [inline README.md]: Yeah that's a great idea. I can submit that as part of the PR

Comment (xiang90) [inline README.md]: Awesome. Thanks!

**Comment by lachie83** [comment]
Good catch. Thanks @rimusz 👍

**Comment by lachie83** [comment]
Thanks team! I appreciate all the support

**Comment by lachie83** [comment]
Added link and deployment. Please review.

**Comment by hongchaodeng** [comment]
LGTM.

Thanks @lachie83

**Pull request merged**